src/map/clif.cpp

```diff
@@ -1701,6 +1702,8 @@ int clif_spawn( struct block_list *bl, bool walking ){
 		clif_spawn_unit( bl, AREA_WOS );
 	}
```

Comment:
```suggestion
	}

	unit_data* ud = unit_bl2ud(bl);
```
Just moved this down since there's the possibility above for checks to fail.